ui/cypress/integration/tests.js

```diff
@@ -11,6 +11,11 @@ describe("Basic tests", () => {
     cy.get("li:Contains(Conditions)").click();
     cy.get("button[role=checkbox]").click();
     cy.get("a[aria-label=back]").click();
+
+    cy.get("button:Contains(Add Criteria)").first().click();
+    cy.get("li:Contains(Race)").click();
+    // required attribute.test.tsx cy.get("button[role=checkbox]").click();
```

Comment:
What does this mean? Why is it a comment?

Reply:
because I do not have the fake content with checkboxes in the attribute page to click. (So, require attribute.test.tsx file)

Reply:
You should add whatever fake data you need to apiContext.ts so that you can write a meaningful test.

Reply:
after communication -- 
add fake data in apiContext.ts within FakeEntitiesApi-listentities-entities